README.md

```diff
@@ -88,14 +88,14 @@ public class HelloWorld {
                         .field(newFieldDefinition()
                                 .type(GraphQLString)
                                 .name("hello")
-                                .staticValue("world"))
+                                .staticValue("world").build())
```

Comment:
As of a [recent PR](https://github.com/graphql-java/graphql-java/commit/be0e7f3ae90a4c53fd4c5dfe43b4d9cff94f59dd), `GraphQLObjectType.Builder#field()` accepts `GraphQLFieldDefinition.Builder` instances. It will call `#build()` for you, so you don't have to do it yourself all over the place. This PR will be in the next release.